include/crow/http_response.h

```diff
@@ -4,7 +4,15 @@
 #include <ios>
 #include <fstream>
 #include <sstream>
+#if defined(_MSC_VER)
+#define _CRT_INTERNAL_NONSTDC_NAMES 1
 #include <sys/stat.h>
+#if !defined(S_ISREG) && defined(S_IFMT) && defined(S_IFREG)
+#define S_ISREG(m) (((m)&S_IFMT) == S_IFREG)
+#endif
+#else
+#include <sys/stat.h>
+#endif
```

Comment:
It looks like `<sys/stat.h>` is being included anyway, why not take it outside the `if` block?

Reply:
As I understand it MSC needs `#define _CRT_INTERNAL_NONSTDC_NAMES 1` before including `<sys/stat.h>`, but the check for the macro needs to be done after the include. Both things are windows only, so I'm not sure, how I can move the include outside the `if`?

Reply:
Well, of course :D I feel dumb now :D